big-int/src/bigint.hpp

```diff
@@ -31,31 +31,18 @@
 #include <iostream>
 #include <string>
 
-namespace libbig
-{
-class largeInt
-{
+namespace libbig {
+class largeInt {
   private:
     bool sign;
     std::string number;
 
   public:
-    largeInt()
-    {
-    }
-
-    largeInt(int)
-    {
-    }
-    largeInt(int64_t)
-    {
-    }
-    largeInt(long long)
-    {
-    }
-    largeInt(std::string)
-    {
-    }
+    largeInt();
+    largeInt(int);
+    largeInt(int64_t);
```

Comment:
Codacy found an issue: [Class 'largeInt' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/ayaankhan98/bigINT/pullRequest?prid=6285291)